### ISSUE TITLE
[Translator] Provider domains configuration option not used in pull/push command

### ISSUE THREAD

**Issue by Florian-B** [symfony/symfony#46177]
### Symfony version(s) affected

5.4.7

### Description

When using Translator component with "Loco" Provider (it could be the case for every providers) the `domains` configuration it not used by `TranslationPushCommand` or `TranslationPullCommand` commands.

### How to reproduce

With the following configuration

```yaml
# config/packages/translation.yaml
framework:
    translator:
        default_path: '%kernel.project_dir%/translations'
        providers:
            loco:
                dsn: '%env(LOCO_DSN)%'
                domains: ['messages', 'emails']
                locales: ['en', 'fr', 'es']

```

The `--help` option on both commands show configured locales as default, but not domains is not used.

```bash
php bin/console translation:push --help
Description:
  Push translations to a given provider.

Usage:
  translation:push [options] [--] [<provider>]

Arguments:
  provider                 The provider to push translations to. [default: "loco"]

Options:
      --force              Override existing translations with local ones (it will delete not synchronized messages).
      --delete-missing     Delete translations available on provider but not locally.
      --domains[=DOMAINS]  Specify the domains to push. (multiple values allowed)
      --locales[=LOCALES]  Specify the locales to push. [default: ["en","fr","es"]] (multiple values allowed)
...
```

The following command will push all domains to the provider instead of `['messages', 'emails']` :  `php bin/console translation:push`

**Comment by plfort** [comment]
I think this issue is solved in 5.4.8 https://github.com/symfony/symfony/pull/45171

**Comment by Florian-B** [comment]
> I think this issue is solved in 5.4.8 #45171

Indeed, thank you! I close this issue.
